packages/react-ai/src/components/AIConversation/views/Controls/PromptControl.tsx

```diff
@@ -111,6 +112,14 @@ export const PromptControl: PromptControl = () => (
   </Container>
 );
 
+export const AutoHidablePromptControl = (): JSX.Element | undefined => {
+  const messages = React.useContext(MessagesContext);
+
+  if (!messages || messages.length === 0) {
+    return <PromptControl />;
+  }
+};
+
 PromptControl.Container = Container;
```

Comment:
Are you intentionally not exposing the Icon/Header/Buttons on `PromptControl`?

Reply:
Ah was just a miss on my part, will add